### PR TITLE
Add function `GetFileContent` to Google Drive

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
resolves https://github.com/wso2-enterprise/choreo/issues/6413

### CODE REVIEW

File: drive/utils.bal [hunk 1]
```diff
@@ -662,3 +665,20 @@ isolated function uploadFileUsingByteArray(http:Client httpClient, byte[] byteAr
...
+# + return - `FileContent` if successful, `error` if not.
+isolated function generateRecordFileContent(http:Client httpClient, string path) returns @tainted FileContent|error {
+    http:Response response = <http:Response> check httpClient->get(<@untainted>path);
+    if (response.statusCode == http:STATUS_OK) {
+        return {
+            content: check response.getBinaryPayload(),
+            mimeType: response.getContentType()
+        }; 
+    } else {
+        fail error(ERR_EXTRACTING_BINARY_PAYLOAD);
```
Comment (abeykoon) [inline drive/utils.bal:681]: Shall we extract error detail from the response. Otherwise we loose information.

File: drive/types.bal [hunk 2]
```diff
@@ -404,6 +405,7 @@ public type FilesResponse record {
 public type GetFileOptional record {
```
Comment (abeykoon) [inline drive/types.bal:405]: If we use this record internally only, shall we not make it public?

File: drive/utils.bal (hunk 1 above)
Comment (SkNuwanTissera) [reply drive/utils.bal:681]: Did the change
Now it's giving the original error as follows
```
                    {
                        "error": {
                        "errors": [
                        {
                        "domain": "global",
                        "reason": "notFound",
                        "message": "File not found: fileId.",
                        "locationType": "parameter",
                        "location": "fileId"
                        }
                        ],
                        "code": 404,
                        "message": "File not found: fileId."
                        }
                        }
                ```

File: drive/types.bal (hunk 2 above)
Comment (SkNuwanTissera) [reply drive/types.bal]: Yeah. It's only used internally. We can change to `public`.
Changed it all places.
/Thanks

**Review by abeykoon — APPROVED** [review]

File: drive/samples/get_file_content_by_id.bal [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+public function main() {
+    drive:Configuration config = {
+        clientConfig: {
+            clientId: clientId,
+            clientSecret: clientSecret,
+            refreshUrl: refreshUrl,
+            refreshToken: refreshToken
+        }
+    };
+    drive:Client driveClient = checkpanic new (config);
```
Comment (indikasampath2000) [inline drive/samples/get_file_content_by_id.bal:43]: Please check why you had to use the `checkpanic`?

Comment (SkNuwanTissera) [reply drive/samples/get_file_content_by_id.bal:43]: Hi @indikasampath2000 , Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/81 
/Thanks

**Pull request merged**